internal/otel/config/helpers/receivers/prometheus_receiver.go

```diff
@@ -40,18 +40,18 @@ func PrometheusReceiverCfg() *PrometheusConfig {
 		  config:
 			scrape_configs:
 			- job_name: consul-telemetry-collector
-			  scrape_interval: 10s
+			  scrape_interval: 1m
 			  static_configs:
 			  - targets:
 				- localhost:9090
 	*/
 
 	return &PrometheusConfig{
 		Config: map[string][]ScrapeConfig{
-			scrapeConfigKey: []ScrapeConfig{
+			scrapeConfigKey: {
 				{
 					JobName:        "consul-telemetry-collector",
-					ScrapeInterval: "10s",
+					ScrapeInterval: "1m",
```

Comment:
This is to make it the same as Consul ? That makes sense to me 👍🏽

Reply:
Kind of ... it's not going to be 1m as the collector will send as soon as it's "bucket" of metrics is full. This is more to scrape itself less.

Reply:
Ah sounds good, thanks for the context 👍🏽